content/en/docs/architecture/component-overview.md

```diff
@@ -183,6 +183,175 @@ return
 
 From this moment on, the device is reachable to all authorized clients and devices. Resource update requests received by a particular Gateway to which the client is connected are forwarded to the [Resource Aggregate](#resource-aggregate). Successful command validation precedes storing and publishing of this event to the [Event Bus](#event-bus) to which the CoAP Gateway is subscribed. If the update request event targets the device hosted by this instance of the CoAP Gateway, an [UPDATE](https://tools.ietf.org/html/rfc7252#section-5.8.2) is forwarded over the authorized TCP channel to the device. The device response is forwarded to the [Resource Aggregate](#resource-aggregate), which issues a resource updated event, updating the resource projection and informing the client that the update was successful.
 
+#### Device Metadata
+
+{{< plantuml id="device-metadata" >}}
+@startuml Sequence
+skinparam backgroundColor transparent
+hide footbox
+
+participant D as "Device"
+participant CGW as "CoAP Gateway"
+participant RA as "Resource Aggregate"
+participant RD as "Resource Directory"
+participant EB as "Event Bus"
+
+group Device has signed in
+   D -> CGW ++: Sign In
+   activate D
+   CGW -> RA++: Update device to online and twin synchronization to out of sync or disabled state
+   RA -> EB: Publish device metadata updated event
```

Comment:
Other diagrams use full event names, please follow the convention. Public DeviceMetadataUpdated event